src/helpers/__tests__/plist.test.ts

```diff
@@ -44,7 +44,7 @@ describe('plist util', () => {
 
     it('throws an error if the file does not exist', () => {
       expect(() => parsePlist('non-existing-file')).toThrow(
-        "ENOENT: no such file or directory, open 'non-existing-file'"
+        /ENOENT: no such file or directory, open '.*non-existing-file'/
```

Comment:
For some reason, only in Windows, Node 20 and 22 now return the full path instead of only the filename: `ENOENT: no such file or directory, open 'D:\\a\\datadog-ci\\datadog-ci\\non-existing-file'`